src/commands/readme.ts

```diff
@@ -141,13 +145,27 @@ USAGE
 
   renderCommand(config: Config.IConfig, c: Config.Command): string {
     this.debug('rendering command', c.id)
-    const title = template({config, command: c})(c.description || '').trim().split('\n')[0]
-    const help = new Help(config, {stripAnsi: true, maxWidth: columns})
+    const HelpClass = getHelpClass(config)
+    const help = new HelpClass(config, {stripAnsi: true, maxWidth: columns})
+
     const header = () => `## \`${config.bin} ${this.commandUsage(config, c)}\``
+
+    const generateCommandHelp = () =>  {
+      const original = process.stdout.write
+      const chunks: string[] = []
+      process.stdout.write = (chunk: string | Buffer | Uint8Array) => {
+        chunks.push(String(chunk))
+        return true
+      }
+      help.showCommandHelp(c, config.topics)
+      process.stdout.write = original
+
+      return chunks.join('')
+    }
+
     return compact([
       header(),
-      title,
-      '```\n' + help.command(c).trim() + '\n```',
+      '```\n' + stripAnsi(generateCommandHelp().trim()) + '\n```',
```

Comment:
We pass stripAnsi to the help class. Why do we have too do it here too?

Reply:
It’s been a while since I last looked at this, but I think it’s because consumers who build upon `HelpBase` can do whatever they want in `showHelp` and `showCommandHelp`. `stripAnsi` is only used in the concrete `Help` implementation.

Reply:
If the `stripAnsi` is in the concert Help, then that implies to me that custom help implementation should also strip if that option is passed in. It's not a big deal to do it here though, it just removes the flexibility if someone didn't want to strip the ansi, for whatever random reason.